pages/review/[lesson].tsx

```diff
@@ -19,14 +19,14 @@ import _ from 'lodash'
 import { SubmissionStatus } from '../../graphql'
 
 type SubmissionDisplayProps = {
-  submissions: SubmissionData[]
+  submissions: Submission[]
 }
 
 const SubmissionDisplay: React.FC<SubmissionDisplayProps> = ({
   submissions
 }) => (
   <div className="submissions-container container p-0">
-    {submissions.map((submission: SubmissionData) => (
+    {submissions.map((submission: Submission) => (
```

Comment:
Don't need the type here, it gets inferred from the array type.

Reply:
Removed.